### PR TITLE
Document segmentations endpoint

### PR DESCRIPTION
**Pull request by timaccelo**
Wrote documentation for the segmentations end point.

### CODE REVIEW

File: source/includes/endpoints/_segmentations.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+```
+
+
+The segmentation object contains the following:
+
+| Field | Type | Description |
+|:-|:-|:-|
+| **id** | unsigned | A unique identifier for the segmentation. |
+| **title** | string | A name for the segmentation. |
+| **link_type** | enum | The object type that the segmentation is related to, may be 'company', 'affiliation' or 'contact'. |
```
Comment (matthewhughes112) [inline source/includes/endpoints/_segmentations.md]: bold tags `**` are used to label default fields, `link_type` (and fields below) are optional

File: source/includes/endpoints/_segmentations.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+| **standing** | enum | The standing of the segmentation, may be one of "unapproved", "approved", "invoiced", "locked", or empty.|
```
Comment (matthewhughes112) [inline source/includes/endpoints/_segmentations.md]: I believe this should only be `active` or `inactive`

File: source/includes/endpoints/_segmentations.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+| **required** | boolean | Either Yes or No, whether this segmentation is required for the object it is against. |
+| **exclusive** | boolean | Either Yes or No, whether this segmentation is limited to one value. |
+| **leaf** | boolean | Either Yes or No, whether this segmentation is limited to only child option.s |
```
Comment (matthewhughes112) [inline source/includes/endpoints/_segmentations.md]: Trailing 's' before final `|`

File: source/includes/endpoints/_segmentations.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+```shell
+curl -X get \ 
+ https://{deployment}.api.accelo.com/api/v0/segmentations \
+  -H 'authorization: Bearer {access_token}'
+```
+
+`GET /segmentations`
+
+This request returns a list of [segmentations](#the-segmentation-object) for the deployment.
```
Comment (matthewhughes112) [inline source/includes/endpoints/_segmentations.md]: There is an existing 'The Segmentation Object' section under companies [here](https://github.com/Accelo/docs/blob/master/source/includes/endpoints/_companies.md#the-segmentation-object) (which actually described a different object) which causes this `#the-segmentation-object` link to be ambiguous, we'll need to chat about how we want to handle this.

File: source/includes/endpoints/_segmentations.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+#### Configuring the Response
+
+##### Pagination
+This request supports all the [pagination](#configuring-the-response-pagination) parameters.
+
+##### Additional Fields and Linked Resource
+This request supports requesting additional fields and linked resources from the [segmentation](#the-skill-object) using
```
Comment (matthewhughes112) [inline source/includes/endpoints/_segmentations.md]: Wrong link `#the-skill-object` here, but probably worth sorting out how to handle links to the segmentation object (see comment above) before handling this.

File: source/includes/endpoints/_segmentations.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+```shell
+curl -X get \ 
+ https://{deployment}.api.accelo.com/api/v0/segmentation{id} \
+  -H 'authorization: Bearer {access_token}' \
+  -H 'Content-Type: application/x-www-form-urlencoded'
+```
+
+`GET /segmentations/{id}`
+
+This request creates and returns a single [segmentation](#the-segmentation-object) specified by its `id`.
```
Comment (matthewhughes112) [inline source/includes/endpoints/_segmentations.md]: > creates
No creation is handled here

File: source/includes/endpoints/_segmentations.md [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+
+#### Configuring the Response
+
+This request supports requesting additional fields and linked items from the [segmentation object](#the-segmentation-object)
+using the [`_fields`](#configuring-the-response-fields) parameter.
+
+#### Handling the Response
+The response will be the single, created [segmentation](#the-segmentation-object) with its default fields and any
```
Comment (matthewhughes112) [inline source/includes/endpoints/_segmentations.md]: As above, incorrect reference to creation

File: source/includes/endpoints/_segmentations.md (hunk 2 above)
Comment (timaccelo) [reply source/includes/endpoints/_segmentations.md]: I think you're correct. Udpated 1142553

File: source/includes/endpoints/_segmentations.md (hunk 1 above)
Comment (timaccelo) [reply source/includes/endpoints/_segmentations.md]: Updated 1142553

File: source/includes/endpoints/_segmentations.md (hunk 3 above)
Comment (timaccelo) [reply source/includes/endpoints/_segmentations.md]: Updated 1142553

File: source/includes/endpoints/_segmentations.md (hunk 5 above)
Comment (timaccelo) [reply source/includes/endpoints/_segmentations.md]: Updated 1142553

File: source/includes/endpoints/_segmentations.md (hunk 6 above)
Comment (timaccelo) [reply source/includes/endpoints/_segmentations.md]: Updated 1142553

File: source/includes/endpoints/_segmentations.md (hunk 7 above)
Comment (timaccelo) [reply source/includes/endpoints/_segmentations.md]: Updated 1142553

File: source/includes/endpoints/_segmentations.md (hunk 4 above)
Comment (matthewhughes112) [reply source/includes/endpoints/_segmentations.md]: fixed with cf5b7a5

**Pull request merged**